src/engine/strat_engine/pool.rs

```diff
@@ -350,22 +348,27 @@ impl Pool for StratPool {
         &'a mut self,
         pool_name: &str,
         fs_uuids: &[FilesystemUuid],
-    ) -> StratisResult<Vec<FilesystemUuid>> {
+    ) -> StratisResult<SetDeleteAction<FilesystemUuid>> {
         let mut removed = Vec::new();
+        let mut asserted = Vec::new();
```

Comment:
It looks like you can remove ```asserted``` here entirely, since ultimately it is not used in the return value.

Reply:
Resolved.